data-pipeline/config.py

```
@@ -50,14 +51,36 @@ class DatabaseConfig:
     )
 }
 
+def get_db_config() -> DatabaseConfig:
+    """Load database configuration from environment variables"""
+    db_url = os.getenv("DATABASE_URL")
+    
+    if db_url and not "localhost" in db_url:
```

Ruff (0.15.6)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)